public/v4/apps/evolution-api.yml

```diff
@@ -0,0 +1,238 @@
+            S3_ENDPOINT: ''
+            S3_REGION: 'us-east-1'
+            S3_USE_SSL: 'false'
+            AUTHENTICATION_API_KEY: '$$cap_appname_authentication_api_key'
+            AUTHENTICATION_EXPOSE_IN_FETCH_INSTANCES: 'false'
+            LANGUAGE: 'en'
+            PROXY_HOST: ''
+            PROXY_PORT: ''
+            PROXY_PROTOCOL: ''
+            PROXY_USERNAME: ''
+            PROXY_PASSWORD: ''
+        volumes:
+            - $$cap_appname-data:/evolution
+        depends_on:
+            - $$cap_appname-redis
+            - $$cap_appname-postgres
+            - $$cap_appname-rabbitmq
+    $$cap_appname-redis:
+        image: redis:latest
+        restart: always
+        volumes:
+            - $$cap_appname-redis-data:/data
+    $$cap_appname-postgres:
+        image: postgres:13
+        restart: always
+        environment:
+            POSTGRES_USER: $$cap_postgres_user
+            POSTGRES_PASSWORD: $$cap_postgres_password
+            POSTGRES_DB: $$cap_postgres_db
+        volumes:
+            - $$cap_appname-postgres-data:/var/lib/postgresql/data
+    $$cap_appname-rabbitmq:
+        image: rabbitmq:3-management
+        restart: always
+        environment:
+            RABBITMQ_DEFAULT_USER: $$cap_rabbitmq_user
+            RABBITMQ_DEFAULT_PASS: $$cap_rabbitmq_password
+        volumes:
+            - $$cap_appname-rabbitmq-data:/var/lib/rabbitmq
+caproverOneClickApp:
+    variables:
+        - id: $$cap_ant_version
+          label: Evolution API Version
+          description: Choose the latest version of Evolution API from https://hub.docker.com/r/atendai/evolution-api/tags
+          defaultValue: latest
+        - id: $$cap_appname_authentication_api_key
+          label: Authentication API Key
+          description: Set a strong API key for authentication
+          defaultValue: $$cap_gen_random_hex(32)
+        - id: $$cap_postgres_user
+          label: PostgreSQL Username
+          description: Set the username for PostgreSQL
+          defaultValue: evolution
+        - id: $$cap_postgres_password
+          label: PostgreSQL Password
+          description: Set a strong password for PostgreSQL
+          defaultValue: $$cap_gen_random_hex(16)
+        - id: $$cap_postgres_db
+          label: PostgreSQL Database Name
+          description: Set the database name for PostgreSQL
+          defaultValue: evolution_db
+        - id: $$cap_rabbitmq_user
+          label: RabbitMQ Username
+          description: Set the username for RabbitMQ
+          defaultValue: evolution
+        - id: $$cap_rabbitmq_password
+          label: RabbitMQ Password
+          description: Set a strong password for RabbitMQ
+          defaultValue: $$cap_gen_random_hex(16)
+    instructions:
+        start: Deploying Evolution API with required services (PostgreSQL, Redis, RabbitMQ)...
+        end: Please allow up to 10 minutes for the setup to complete. Once finished, HTTPS and WebSocket support will be activated.
```

Comment:
> Once finished, HTTPS and WebSocket support will be activated

I assume you wanted to say this instead:
Once finished, you need to activate HTTPS and WebSocket support before being able to use the app.